diagnosis/getkuberneteslogs.sh

```diff
@@ -23,6 +23,151 @@ function restore_ssh_config
     fi
 }
 
+retrycmd_if_failure() 
+{ 
+    retries=$1; 
+    wait=$2; 
+    for i in $(seq 1 $retries); do 
+        ${@:3}; [ $? -eq 0  ] && break || sleep $wait; 
+    done; 
+    log_level -i "Command Executed $i times."; 
+}
+
+
+function login_azs()
```

Comment:
I'll rather remove `function` for consistency and because akse scripts do it that way